.vib/keycloak/goss/goss.yaml

```diff
@@ -0,0 +1,29 @@
+file:
+ /opt/bitnami/keycloak/standalone/configuration:
+   exists: true
+   filetype: directory
+   mode: "0775"
+   owner: root
+/opt/bitnami/keycloak/domain/configuration/application-users.properties:
+   exists: true
+   filetype: file
+   mode: "0775"
+   contains:
+   - admin
+/opt/bitnami/scripts/keycloak-env.sh:
+   exists: true
+   filetype: file
+   mode: "0664"
+   owner: root
+command:
+  user-id-test:
```

Comment:
Remember to update this test to check for non-root as discussed.

Reply:
Changed this test to what you did in another PR.